lib/ddp.js

```diff
@@ -23,6 +21,47 @@ const PUBLIC_EVENTS = [
 ];
 const DEFAULT_RECONNECT_INTERVAL = 10000;
 
+class EventInterface {
```

Comment:
Don't remember if I did this or not, but this file is definitely important for reactivity